be/src/storage/lake/filenames.h

```diff
@@ -85,6 +85,10 @@ inline std::string random_segment_filename() {
     return name;
 }
 
+inline std::string schema_filename(int64_t schema_id) {
```

Comment:
Because S3 has parallel limitation on common object's prefix. Why not make schema as postfix of object name?

Reply:
> Prefixes are considered to be the whole path (up to the last '/') of an object's location, and are no longer hashed only by the first 6-8 characters. Therefore it would be enough to just split the data between any two "folders" to achieve x2 max requests per second. (if requests are divided evenly between the two)

https://stackoverflow.com/questions/52443839/s3-what-exactly-is-a-prefix-and-what-ratelimits-apply

Reply:
The full object path is prefixed with table id.